frontend/app/src/pages/main/v1/workflows/$workflow/components/trigger-workflow-form.tsx

```diff
@@ -64,6 +64,29 @@ export function TriggerWorkflowForm({
     defaultWorkflow?.metadata.id,
   );
 
+  const [workflowSearch, setWorkflowSearch] = useState<string>('');
+  const [debouncedWorkflowSearch, setDebouncedWorkflowSearch] =
+    useState<string>('');
+
+  const debouncedSetSearch = useMemo(
+    () => debounce((value: string) => setDebouncedWorkflowSearch(value), 300),
+    [],
```

Comment:
I think this will probably fail the linter - it'll need some dependencies I think

Reply:
🤷 odd, works for me though!